libs/slides/src/lib/shortcuts/shortcuts.directive.ts

```diff
@@ -1,25 +1,29 @@
-import { Directive, HostListener, Optional } from '@angular/core';
+import { Directive, HostListener, Optional, ContentChild } from '@angular/core';
 import { SlidesDeckComponent } from '../deck/deck.component';
+import { SlidesArrowsComponent } from '../arrows/slides-arrows.component';
 import { FullScreenModeService } from '@codelab/utils';
 
 @Directive({
   selector: '[slideShortcuts]'
 })
 export class ShortcutsDirective {
+
+  @ContentChild(SlidesArrowsComponent, { static: false }) private arrows: SlidesArrowsComponent;
```

Comment:
Hm, instead of requiring arrows and being tightly bound to them, can we just have an attribute on the DOM element and check if target has this dom element?

I can totally forsee having a nicer looking arrows component, or wanting to have this functionality for progress bar